apps/root/src/services/conctactListService.ts

```diff
@@ -69,6 +80,55 @@ export default class ContactListService implements IContactListService {
     }
   }
 
+  async transferTokenToContact({
```

Comment:
I'm thinking this method should actually go into the `walletService` since it does not matter if the recipient is a contact or not